modules/dr-hosted-cluster-within-aws-region-delete.adoc

```diff
@@ -21,6 +21,13 @@ Ensure that the `kubeconfig` file of the destination management cluster is place
 
 . Scale the `deployment` and `statefulset` objects by entering these commands:
 +
+[IMPORTANT]
+====
+Do not scale the stateful set if the value of its `spec.persistentVolumeClaimRetentionPolicy.whenScaled` field is set to `Delete`, because this could lead to loss of data.
```

Comment:
```suggestion
Do not scale the stateful set if the value of its `spec.persistentVolumeClaimRetentionPolicy.whenScaled` field is set to `Delete`, because this could lead to a loss of data.
```